### PR TITLE
Bump to 0.3.35

### PR DESCRIPTION
**Pull request by guillochon**

### CODE REVIEW

**Comment by guillochon** [comment]
Hey @CJ-Wright, having a bit of difficulty with this update; conda is complaining about needing pip to install a package, but the error log does not specify *which* package for which this is so. Also, the dependencies are the same as the last time the package was built successfully, so it's not clear why there's a change. Any ideas?

**Comment by guillochon** [comment]
Think I found it, missed that the package URL was listed after the error message.

**Comment by CJ-Wright** [comment]
needs `secretstorage`

**Comment by guillochon** [comment]
Thanks @CJ-Wright, looks like this package is actually imported by another package, but I don't know which one (suggests one of the packages we depend on doesn't have the correct deps). But I guess I'll add it to the meta anyway.

**Comment by CJ-Wright** [comment]
I'd iterate this on your local machine with conda build, that way you don't need to wait for the CI to give back the answer. Also please note that most of these are most likely not build requirements.

**Comment by guillochon** [comment]
Sadly the CI is faster than my own machine, I don't know if it's because there's some caching done at the CI, but it takes me ~10 minutes to run a local build.

I think the build should require a select few packages but I am not sure.

**Comment by CJ-Wright** [comment]
You might also take a look at the `setup.py` to find the packages specified there.

**Comment by CJ-Wright** [comment]
@conda-forge-admin, please rerender

**Comment by CJ-Wright** [comment]
here are the requirements https://github.com/astrocatalogs/astrocats/blob/master/requirements.txt

File: recipe/meta.yaml [hunk 1]
```diff
@@ -12,45 +12,38 @@ source:
...
     - beautifulsoup4
     - bokeh
     - future
-    - gitdb2
     - gitpython
     - inflect
-    - keyring
     - palettable
     - psutil
     - seaborn
```
Comment (CJ-Wright) [inline recipe/meta.yaml:32]: I don't think these are all build requiements

**Comment by guillochon** [comment]
Whew finally! Thanks for the help @CJ-Wright!

**Pull request merged**